src/client/components/form-elements/ReviewCollapsiblePanel.jsx

```diff
@@ -52,16 +52,21 @@ class ReviewCollapsiblePanel extends React.Component {
       panelAction = (<ErrorableCheckbox
           label="I certify that all information above is correct to the best of my knowledge."
           checked={this.props.value}
+          additionalClass={this.props.additionalClass}
```

Comment:
FYI this is going to change because we're removing the checkboxes and replacing them with "Next" buttons.